ci_framework/roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -105,6 +105,16 @@
     index_var: vm_id
     label: "{{ vm_type }}-{{ vm_id }}"
 
+- name: Attach listed networks to the VMs
+  vars:
+    vm_item: "{{ vm_type }}-{{ vm_id }}"
+    networks: "{{ vm_data.value.nets }}"
+  ansible.builtin.include_tasks: net_to_vms.yml
+  loop: "{{ range(0, vm_data.value.amount | default(1) | int) }}"
+  loop_control:
```

Comment:
How nice is ansible to just loop over a loop...

Reply:
:) at least it's possible. Imagine if it wasn't....